examples/twittermap/web/public/javascripts/common/services.js

```diff
@@ -81,7 +84,7 @@ angular.module('cloudberry.common', [])
       }
     }
 
-    function getFilter(parameters, maxDay) {
+    function getFilter(parameters, maxDay, geoIds) {
```

Comment:
what's the purpose of the `geoIds` here? it is never used?

Reply:
It has been added to get filter with all geoIds in case of complete cache miss and only new geoIds in case of partial cache miss. It's used by byGeoRequest().